core/modules/hubot/responder.js

```diff
@@ -0,0 +1,24 @@
+﻿var http = require('scoped-http-client');
```

Comment:
![Codacy](https://www.codacy.com/assets/images/favicon.png) Issue found: [Irregular whitespace not allowed](https://www.codacy.com/app/theprojectshop/Kassy/file/2530333297/issues/source?bid=3282987&fileBranchId=3282997#l1)